modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -30,6 +30,21 @@
 
 The following premium plugin updates were released alongside {productname} 6.5.
 
+=== Advanced Code Editor 3.3.0
+
+The {productname} 6.5 release includes an accompanying release of the **Advanced Code Editor** premium plugin.
+
+**Advanced Code Editor** 3.3.0 includes the following fix.
+
+=== The `copy code` button wasn't correctly styled within the Advanced Code Editor.
+//#TINY-9698
+
+In previous versions of {productname}, the "Copy code" button had a grey background.
```

Comment:
there were no previous implementation of this button, is just that the first implementation was wrongly styled

Reply:
@lorenzo-pomili please see below suggestion